labelspark/__init__.py

```diff
@@ -1,6 +1,5 @@
 import json
 import urllib
-import databricks.koalas as pd
```

Comment:
We can use a conditional check here to avoid breaking LTS installs: 

```
from packaging import version

if version.parse(spark.version) < version.parse("3.2.0"): 
  import databricks.koalas as pd 
  needs_koalas = True
else: 
  needs_koalas = False
```